test/jdk/java/security/Signature/TestCloneable.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file, including the copyright date and the final newline, should be identical to upstream.

Reply:
Updated